docs/topics/returns.md

```diff
@@ -39,32 +41,42 @@ loop@ for (i in 1..100) {
 A `break` qualified with a label jumps to the execution point right after the loop marked with that label.
 A `continue` proceeds to the next iteration of that loop.
 
-## Return to labels
+### Non-local break and continue
 
-In Kotlin, functions can be nested using function literals, local functions, and object expressions.
-Qualified `return`s allow us to return from an outer function.
-The most important use case is returning from a lambda expression. Recall that when we write the following,
-the `return`-expression returns from the nearest enclosing function - `foo`:
+> This feature is currently [Experimental](components-stability.md#stability-levels-explained).
+> We're planning to stabilize it in future releases.
+> To opt in, use the `-Xnon-local-break-continue` compiler option.
+> We would appreciate your feedback on it in [YouTrack](http://kotl.in/issue).
```

Comment:
It's not clear to me how much sense it makes to point to the YouTrack new issue page in such kind of sentence. At least, it's slightly unexpected.

Maybe a better option would be to point to the issue itself https://youtrack.jetbrains.com/issue/KT-1436